apps/dialtone-documentation/docs/about/whats-new/posts/2024-10-3.md

```diff
@@ -0,0 +1,65 @@
+---
+heading: Dialtone Vue is now fully tree-shakeable (BREAKING CHANGE)
+author: Nina Repetto
+posted: '2024-10-3'
+---
+
+<BlogPost :author="$frontmatter.author" :posted="parse($frontmatter.posted, 'y-M-d', new Date())" :heading="$frontmatter.heading">
+
+Hello there! We're excited to share that we've enhanced the tree-shaking capabilities of Dialtone components. This allows you to import just the components you require, rather than the entire library. This change will help decrease your bundle size and enhance performance.
+
+You can read more about how to take advantage of this feature in [this blog post](/about/whats-new/posts/2024-4-15.html).
+
+There are some breaking changes to be aware of in Dialtone version 9.77.0. Please update your projects as mentioned below. Dialpad and Dialpad Meetings were already updated, so the following instructions are for other projects using Dialtone.
+
+## Breaking changes
```

Comment:
We may want to state that the old dt-icon component is still available, however if you use it, it will import all icons instead of just the icons being used.